homeassistant/components/climate/ecobee.py

```diff
@@ -145,12 +145,30 @@ def current_temperature(self):
     @property
     def target_temperature_low(self):
         """Return the lower bound temperature we try to reach."""
-        return int(self.thermostat['runtime']['desiredHeat'] / 10)
+        if self.operation_mode == 'auto':
```

Comment:
This one is still here. :wink:
If you want, you can also import `STATE_AUTO` from the climate component and use that instead of `'auto'`.